### DISCUSSION TITLE
SCI Inputs on Shared Infra

### DISCUSSION THREAD

**Discussion by navveenb** [Green-Software-Foundation/sci#249] [General]
Got this query from Yugal Joshi (Partner, everestgrp.com) -  "On Software Carbon Intensity formula - I didn’t catch how the energy consumed by software in KwH and carbon generated by hardware where SW is running, is calculated here.
 
Reason I ask is many clients run shared infra and getting to apps-specific HW carbon consumption is very challenging, almost nearly impossible. But calculating energy consumed by SW (if we take our HW as that’s is separate in your equation) is something I am more curios about."

More context -
"I understand how to calculate energy consumed by HW where the SW is hosted, this part I am more or less clear about. What I am less clear is how are we calculating energy consumed by SW."

**Comment by atg-abhishek** [comment]
@navveenb can you add more context around who Yugal is and his background so that our responses can be tailored so that he may understand how we're approaching this better?

**Reply by navveenb** [reply]
Sure, 

More updates on this thread -



> @navveenb can you add more context around who Yugal is and his background so that our responses can be tailored so that he may understand how we're approaching this better?

Sure, I edited the original discussion to add the context
